Exploring_Pegasus/get_inactive_qubits.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import dwave_networkx as dnx
+from dwave.system.samplers import DWaveSampler
+
+dwave_sampler_pegasus = DWaveSampler(solver={'topology__type': 'pegasus'})
+props_pegasus = dwave_sampler_pegasus.properties
+
+# Get total qubits - should be 24 * N * (N - 1)
+total_qubits = props_pegasus['num_qubits']
+
+# Get total number of inactive qubits
+total_inactive = [i for i in range(total_qubits) if i not in dwave_sampler_pegasus.nodelist]
```

Comment:
For posterity, this should work as well:
```
num_qubits = dwave_sampler_pegasus.solver.num_qubits
num_active_qubits = dwave_sampler_pegasus.solver.num_active_qubits
total_inactive = num_qubits - num_active_qubits
```
To get a list of inactive:
```
inactive_qubits = set(range(props_pegasus['num_qubits'])).difference(props_pegasus['qubits'])
```
or via `dwave.cloud.solver.Solver`:
```
solver = dwave_sampler_pegasus.solver
inactive_qubits = set(range(solver.num_qubits)) - solver.nodes
```